src/App/pages/LandingPage/Main/FilterFormCinema/FilterFormCinema.js

```diff
@@ -0,0 +1,35 @@
+export function FilterFormCinema({}) {
+    let cinemaData =[{"id":655,"cinemaName":"eiufeiufheuf","adress":{"streetAddress":"6388 Lacus Ct","city":"Kenora"}},
+    {"id":761,"cinemaName":"qsqsq","adress":{"streetAddress":"6057 Vitae Rd","city":"Washington"},},
```

Comment:
The comma after the address value is redundant since you don't wrap the object to the next line. 
Either wrap it like this or remove the comma. 
{ 
    id: 761,
    cinemaName: "qsqsq",
    adress: { streetAddress: "6057 Vitae Rd", city: "Washington" },
 },

Reply:
Also, do not use quotes around properties keys. There are some rare cases when you can do this (e.g, if your property name should have multiple words separated by spaces like  "Apple Tea"), but otherwise don't. It makes object literals less readable.

Reply:
fixed